strapi/strapi
# Settings: Move SSO page into pages directory

### What does it do?

- Moves the SSO page into the pages directory (it was sitting outside)
- Cleanup SSO routes to be easier to extend with the next feature
- Cleanup tests (remove inline snapshot test and test for particular strings instead)

### Why is it needed?

Cleanup.

### How to test it?

1. Run Strapi in EE mode and verify the SSO page is rendered
2. Run Strapi in CE mode and verify the SSO page is not rendered
